packages/internal/src/routes.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs'
+
+import chalk from 'chalk'
+
+import { getPaths } from './paths'
+
+export interface Route {
+  name?: string
+  path?: string
+  page?: string // All valid routes should have a page but it's optional here for convenience
```

Comment:
The comment isn't entirely true. Redirect routes don't have a page

Reply:
Ah true indeed, removed.